packages/node-cli/src/commands/object/update/UpdateInputHandler.js

```diff
@@ -41,7 +48,48 @@ module.exports = class UpdateInputHandler extends BaseInputHandler {
 		this._fileSystemService = new FileSystemService();
 	}
 
-	async getParameters(params) {
+	async getParameters(_params) {
```

Comment:
This was a VsCode suggestion, because params is not used.
Normally in Javascript, you put the varaibles you don't use (but you need to put because of inheritance) with an underscore.
Anyways, I could put it params normally